### PR TITLE
Run tests and doc build for 23.1 docker

### PR DESCRIPTION
**Pull request by mkundu1**
Only the 22.2 artifacts have been uploaded. ~One concern could be that the CI log is showing all settings API filenames (we should hide any 23.1 solver details).~

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -194,7 +194,7 @@ jobs:
       - name: Install again after codegen
         run: |
           rm -rf dist
-          make install
+          make install > /dev/null
```
Comment (mkundu1) [inline .github/workflows/ci.yml:197]: To suppress logging settings API filenames

**Comment by mkundu1** [comment]
@akulshre-qa We need to make sure 23.1 specific tests are running with 23.1 image only (possibly using a marker).

**Comment by akulshre-qa** [comment]
> @akulshre-qa We need to make sure 23.1 specific tests are running with 23.1 image only (possibly using a marker).

Do we need to make changes in makefile to do this?

**Review by akulshre-qa — APPROVED** [review]

**Comment by mkundu1** [comment]
> > @akulshre-qa We need to make sure 23.1 specific tests are running with 23.1 image only (possibly using a marker).
> 
> Do we need to make changes in makefile to do this?

Yes, I've created #681 to cover it.

**Review by seanpearsonuk — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

**Pull request merged**